### DISCUSSION TITLE
database.db user

### DISCUSSION THREAD

**Discussion by florentin68** [Koenkk/zigbee2mqtt#6531] [General]
Hello,
My automatic update script fails because `database.db` file in `/data` is owned by another user (named `jeedom`) than the rest of the z2m install (`www-data`).
When I change the ownership and permissions for this file, this is reset when I restart z2m.
Why is the owner of this file different than other files in this directory?
Thank you in advance for your help.

**Comment by git4ndre** [comment]
I guess your zigbee2mqtt.service (daemon) is running as user `jeedom`.

**Reply by m0wlheld** [reply]
Or - the Zigbee2MQTT service uses a UID that is assigned to the local user jeedom. Did you check the UID?

**Reply by florentin68** [reply]
> I guess your zigbee2mqtt.service (daemon) is running as user `jeedom`.

You're right.
Wrong user was configured in `/etc/systemd/system/zigbee2mqtt.service`.
Thank you for your help!
